Gaijinizer patch/Map308.txt

```diff
@@ -150,9 +148,10 @@ The timing is still in a coma, but......#GT
 
 可能性は高いだろうさ。
 > CONTEXT: Map308/events/5/pages/0/78/Dialogue/other1 
-Well, I'm sure you're taking part in the
-evacuation guide, so maybe I'm bringing you
-to a shelter. The chances are high.#GT
+Well, if Yuuma's working evacuation services,
+it's probably to keep you safe in the shelter...
+
+The odds are pretty high.#Amaboye
 > END STRING
```

Comment:
![image](https://user-images.githubusercontent.com/48231102/126911181-88b684ec-5900-4c80-9110-cc1b30856d86.png)

Is this interpretation for the first part correct?

Reply:
from our talk in the server:
```
Well, if Yuuma's part of the evacuation, they're
probably taking you to the shelter right now...

The odds are pretty high.#Amaboye
```